.github/workflows/release-singleimage.yml

```diff
@@ -24,17 +24,93 @@ jobs:
       - name: "Checkout"
         uses: actions/checkout@v2
         with:
+          repository: Budibase/budibase
+          ref: 'master'
           submodules: true
           token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
           fetch-depth: 0
-
       - name: Fail if tag is not in master
         run: |
           if ! git merge-base --is-ancestor ${{ github.sha }} origin/master; then
             echo "Tag is not in master. This pipeline can only execute tags that are present on the master branch"
             exit 1
           fi
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
I think we should have an `_inc` file for the actual build and release, which accepts an arg for the platforms and image name, so that we don't need to duplicate the configuration multiple times as that could be bug prone

Reply:
I can see pros and cons with that as we may also want to remove some of the buildx steps from some jobs. I'd like to see this running and I'll create a ticket to add refinements.